src/components/axes/timeAxis.ts

```diff
@@ -1,6 +1,11 @@
 ///<reference path="../../reference.ts" />
 
 module Plottable {
+
+export enum TimeInterval {
+  second, minute, hour, day, week, month, year
```

Comment:
yeah, but that's not going to be an enum anymore. Can do something like this http://stackoverflow.com/questions/15490560/create-an-enum-with-string-values-in-typescript